docker/pennylane.dockerfile

```diff
@@ -46,7 +46,7 @@ RUN pip install pytest pytest-cov pytest-mock flaky
 RUN pip install -i https://test.pypi.org/simple/ pennylane-lightning --pre --upgrade
 # hotfix, remove when pyscf 2.1 is released (currently no wheel for python3.10)
 RUN pip install openfermionpyscf || true
-RUN pip install dill zstd || true
+RUN pip install hdf5 || true
```

Comment:
Are the other two libraries optional here as well?

Reply:
ah, probably not - added them here